timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -0,0 +1,96 @@
+"""Index analyzer plugin for sigma."""
+from __future__ import unicode_literals
+import os
+
+from sigma.backends import elasticsearch as sigma_elasticsearch
+import sigma.configuration as sigma_configuration
+from sigma.parser import collection as sigma_collection
+
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class SigmaPlugin(interface.BaseIndexAnalyzer):
```

Comment:
Note: I think we want to move this to a SketchAnalyzer later, so it can be run from a sketch as well as creating Views etc. This can be changed later though seeing the only action in this version is tagging.

Reply:
Changed this to a sketch analyzer, thought I think we'll want to think more about this in the future. If we use rules that contain aggregations for example, we don't necessarily want to aggregate over the whole sketch if contains data from multiple machines.

Reply:
Ah, yeah. So you can choose what timelines to operate on, so it's totally fine to just run on the current timeline (this is the default).